src/System.Net.Security/src/System.Net.Security.csproj

```diff
@@ -387,6 +387,9 @@
     <Compile Include="$(CommonPath)\Microsoft\Win32\SafeHandles\Asn1SafeHandles.Unix.cs">
       <Link>Common\Microsoft\Win32\SafeHandles\Asn1SafeHandles.Unix.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\Microsoft\Win32\SafeHandles\SafeHandleCache.cs">
+      <Link>Common\Microsoft\Win32\SafeHandles\SafeHandleCache.cs</Link>
+    </Compile>
```

Comment:
Why are these additions to System.Net.Security and System.Net.Http necessary?

Reply:
To use SafeHandlesCache in Interop.X509Stack.